include/swift/Runtime/Metadata.h

```diff
@@ -789,7 +789,7 @@ void swift_registerTypeMetadataRecords(const TypeMetadataRecord *begin,
 /// Return the superclass, if any.  The result is nullptr for root
 /// classes and class protocol types.
 SWIFT_CC(swift)
-SWIFT_RUNTIME_STDLIB_INTERNAL
+SWIFT_RUNTIME_EXPORT
```

Comment:
Should be `SWIFT_RUNTIME_STDLIB_SPI`. Otherwise, seems like a reasonable change to me.

Reply:
Ah, okay - I'll adjust that!